src/composer.json

```diff
@@ -0,0 +1,75 @@
+{
+    "name": "blumilksoftware/example-app",
+    "type": "project",
+    "description": "example-app project.",
+    "keywords": ["framework", "laravel"],
+    "license": "MIT",
+    "require": {
+        "php": "^8.2",
+        "guzzlehttp/guzzle": "^7.7.0",
+        "inertiajs/inertia-laravel": "^0.6.9",
+        "laravel/framework": "^10.13.0",
+        "laravel/sanctum": "^3.2.5",
+        "laravel/tinker": "^2.8.1",
+        "ext-pdo": "*"
+    },
+    "require-dev": {
+        "blumilksoftware/codestyle": "^2.3.0",
+        "fakerphp/faker": "^1.22.0",
+        "mockery/mockery": "^1.5.1",
+        "nunomaduro/collision": "^7.5.2",
+        "nunomaduro/larastan": "^2.0",
+        "phpunit/phpunit": "^10.1.3",
+        "spatie/laravel-ignition": "^2.1.3"
+    },
+    "autoload": {
+        "psr-4": {
+            "ExampleApp\\": "app/",
+            "Database\\Factories\\": "database/factories/",
+            "Database\\Seeders\\": "database/seeders/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/"
+        }
+    },
+    "scripts": {
+        "post-autoload-dump": [
+            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
+            "@php artisan package:discover --ansi"
+        ],
+        "post-update-cmd": [
+            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
+        ],
+        "post-root-package-install": [
+            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
+        ],
+        "post-create-project-cmd": [
+            "@php artisan key:generate --ansi"
+        ],
+        "test": [
+            "@putenv XDEBUG_MODE=off",
+            "@php artisan test"
+        ],
+        "analyse": "./vendor/bin/phpstan analyse",
+        "cs": "./vendor/bin/php-cs-fixer fix --dry-run --diff --config codestyle.php",
+        "csf": "./vendor/bin/php-cs-fixer fix --diff --config codestyle.php"
+    },
+    "extra": {
+        "laravel": {
+            "dont-discover": []
+        }
+    },
+    "config": {
+        "optimize-autoloader": true,
+        "preferred-install": "dist",
+        "sort-packages": true,
+        "allow-plugins": {
+            "pestphp/pest-plugin": true,
+            "php-http/discovery": true
+        }
+    },
+    "minimum-stability": "stable",
+    "prefer-stable": true
+}
```

Comment:
I would remove composer.lock from this repo.

Reply:
package-lock.json too.

Reply:
Should I add this to `.gitignore`? or not?

Reply:
removed